pulser/simulation/simulation.py

```diff
@@ -44,28 +46,25 @@ class Simulation:
         sampling_rate (float): The fraction of samples that we wish to
             extract from the pulse sequence to simulate. Has to be a
             value between 0.05 and 1.0
-        evaluation_times (str,list): The list of times at which the quantum
+        evaluation_times (str,list,float):
+            The list of times at which the quantum
             state should be evaluated, in μs. If 'Full' is provided, this list
             is set to be the one used to define the Hamiltonian to the solver.
             The initial and final times are always included, so that if
             'Minimal' is provided, the list is set to only contain the initial
-            and the final times.
+            and the final times. Use a float to act as a sampling rate for
+            the resulting state.
     """
 
     def __init__(self, sequence: Sequence, sampling_rate: float = 1.0,
```

Comment:
Just an idea here, open for discussion. 

The Simulation class is getting big. What about separating from the class all the configs elements related to noise ? Then, we could provide the necessary elements at initialization. Something like my_sim = Simulation(sequence, noise_config).

Reply:
I added a new `SimConfig` class in an attempt to be clearer. It encapsulates noise types, temperature field, number of runs, and solver options.